src/game_pictures.cpp

```diff
@@ -124,6 +124,10 @@ int Game_Pictures::GetDefaultNumberOfPictures() {
 		return 1000;
 	}
 	else if (Player::IsMajorUpdatedVersion()) {
+		if (Player::IsPatchDynRpg() && Player::IsRPG2k3()) {
+			// DynRPG only exists for RPG Maker 2003 1.08
+			return 2000;
```

Comment:
If you don't mind: I added a commit that allocates 2000 pic slots for savegame compat.